src/components/Footer.tsx

```diff
@@ -0,0 +1,64 @@
+import { Todo } from '../types/Todo';
+import cn from 'classnames';
+
+type Props = {
+  todos: Todo[];
+  filter: 'all' | 'active' | 'completed';
+  handleFilter: (selectedFilter: 'all' | 'active' | 'completed') => void;
+};
+
+export const Footer: React.FC<Props> = ({ todos, filter, handleFilter }) => {
+  return (
+    <>
+      {/* Hide the footer if there are no todos */}
+      <footer className="todoapp__footer" data-cy="Footer">
+        <span className="todo-count" data-cy="TodosCounter">
+          {todos.filter(todo => !todo.completed).length} items left
```

Comment:
Create separate variable for this `todos.filter(todo => !todo.completed).length`